refinery/data_set_manager/utils.py

```diff
@@ -722,6 +730,21 @@ def generate_solr_params(params, assay_uuids, facets_from_config=False):
     return encoded_solr_params
 
 
+def cull_attributes_from_list(attribute_list, remove_facets):
```

Comment:
`facets_to_remove` instead of `remove_facets`?

Also, are we culling attributes or removing facets here? Maybe rename the function if appropriate